src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -186,16 +225,15 @@ def download_binary_dsn(self, dataset_name, output_file, with_prefixes=False):
         bytes
             Binary content of the dataset.
         """
-        content = self.get_dsn_binary_content(dataset_name, with_prefixes=with_prefixes)
+        content = self.get_dsn_binary_content_streamed(dataset_name, with_prefixes=with_prefixes)
         out_file = open(output_file, 'wb')
         out_file.write(content)
         out_file.close()
 
     def upload_file_to_dsn(self, input_file, dataset_name, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
         """Upload contents of a given file and uploads it to a dataset."""
         if os.path.isfile(input_file):
-            in_file = open(input_file, 'r')
-            file_contents = in_file.read()
-            response_json = self.write_to_dsn(dataset_name, file_contents)
+            with open(input_file, 'r') as file_contents:
+                response_json = self.write_to_dsn(dataset_name, file_contents)
```

Comment:
Using the variable name `in_file` would be better here instead of `file_contents`, since we are no longer reading the contents of the file but now passing the file object on to `requests`.